server/app/uk/gov/ons/addressIndex/server/controllers/AddressController.scala

```diff
@@ -81,7 +81,7 @@ class AddressController @Inject()(val controllerComponents: ControllerComponents
     val lonVal = lon.getOrElse("")
 
     val epochVal = epoch.getOrElse("")
-    val fromsourceVal = fromsource.getOrElse("all")
+    val fromsourceVal = {if (fromsource.getOrElse("all").isEmpty) "all" else fromsource.getOrElse("all")}
```

Comment:
This can be shortened using Scala to `fromsource.filter(!_.isEmpty).getOrElse("all")`. The braces are not needed.